### ISSUE TITLE
Cannot use sequelize transaction with the default logger hook

### ISSUE THREAD

**Issue by zaro** [feathersjs/generator-feathers#357]
in generator-feathers/generators/app/templates/src/hooks/logger.js there is this debug print :

```javascript
    if(typeof context.toJSON === 'function') {
      logger.debug('Hook Context', JSON.stringify(context, null, '  '));
   }
```
If I try to use transaction with sequelize ( feather-sequelize ) the debug statement fails with :

error:  TypeError: Converting circular structure to JSON

Simple fix would be:

```javascript
const util = require('util');
...
    if(typeof context.toJSON === 'function') {
      logger.debug('Hook Context', util.inspect(context));
    }
```

**Comment by daffl** [comment]
Makes sense. If you submit a pull request we can get it into the next release 😄

**Comment by zaro** [comment]
Cool:) Here it is #358

**Comment by daffl** [comment]
Closed via #358
